### PR TITLE
[ESOMAR.org] Update and reactivate ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/ESOMAR.org.xml [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+	Login required:
+		cloud.esomar.org
+		library.esomar.org
+		tracker.esomar.org
+
+	Different content http/https:
+		jobs.esomar.org
+		my.esomar.org
+
+	Mixed content:
```
Comment (J0WI) [inline src/chrome/content/rules/ESOMAR.org.xml]: Any examples?

Comment (Bisaloo) [inline src/chrome/content/rules/ESOMAR.org.xml]: I have mixed content from `wyldedragonne.com` and `connect.facebook.com` on https://rwconnect.esomar.org/ with Chromium.

Comment (J0WI) [inline src/chrome/content/rules/ESOMAR.org.xml]: Confirmed, thanks!

**Comment by Bisaloo** [comment]
#9842

**Pull request merged**